azure_encryption_helper/encryptor.py

```diff
@@ -0,0 +1,47 @@
+from .aescipher import AESCipher
+from .aeskeywrapper import AESKeyWrapper
+
+import os
+
+class Encryptor:
+
+    def __init__(self, key=None, iv_length=16):
+        if key is None:
+            key = Encryptor.generate_aes_key()
+        self.key = key
+        self.iv_length = iv_length
+        self._update_cipher()
+
+    @staticmethod
+    def generate_aes_key(length=32):
+        return os.urandom(length)
+
+    def configure_wrapper(self, vault, credentials, key_name, key_version):
```

Comment:
Do we need to be able to set the wrapper or can we just do this as part of `__init__`

Reply:
Agreed, when will there be a case when you do not need to configure the wrapper?

Reply:
You're not required to bring KV if you already get AES key somehow.
Just so users can use same library regardless of their KV needs.